docs/2fa/users/guide.md

```diff
@@ -0,0 +1,100 @@
+# User Guide
+
+<!-- Visit [Automata 2FA Guru](https://app.2fa.guru) to start your 2FA protection. -->
+Visit Automata 2FA Guru to start your 2FA protection.
+
+---
+
+## Setup 2FA
+<!-- - Visit [Automata 2FA Guru](https://app.2fa.guru). -->
+- Visit Automata 2FA Guru.
+
+- Use metamask and connect your wallet.
+
+- If your account hasn't signed up, you can click the **Wallet Actions** button on the top, and go through the signup process. You need to sign a message to prove your identity before generating a 2FA secret. Attention, you must use a different device to store the 2FA secret, don't keep the 2FA device and your wallet in the same one.
```

Comment:
If you haven't signed up for an account, you can click the **Wallet Actions** button at the top to initiate the process of signing up. You will be required to sign a message to prove your identity before generating a 2FA secret. Take note, the primary wallet and 2FA device should be different devices."